app/models/user.rb

```diff
@@ -2,4 +2,16 @@ class User < ApplicationRecord
   include GDS::SSO::User
   serialize :permissions, Array
   RESOURCE_MANAGER_PERMISSION = 'resource_manager'.freeze
+  GUIDER_PERMISSION = 'guider'.freeze
+
+  def self.guiders
+    # This can't really be made faster because we're storing
+    # permissions as a serialized string. We only really should have a maximum of
+    # 50 (guiders) + 3 (resource managers) in the database though.
+    User.all.select { |u| u.permissions.include? GUIDER_PERMISSION }
```

Comment:
If we expose a `guider?` predicate like you've done for resource managers we could golf this down to `User.all.select(&:guider?)`